### PR TITLE
#163519103 Implement article's rating

### PR DESCRIPTION
**Pull request by oesukam**
#### What does this PR do?
Implement article's rating

#### Description of Task to be completed?
- Add article's rating logic in view a single article page
- Create a page to view all article'ratings

#### How should this be manually tested?
- Clone the repo
- Run `npm install`
- Run `npm start`
- Login as a registered user
- Navigate to view a single article using the following url's format `/articles/<articleSlug>`
- Scroll down to apply your rating

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
`#163519103`

#### Screenshots
- View a single article page
image

- Article's ratings
image

#### Questions:
N/A

### CODE REVIEW

File: src/__tests__/reducers/articleReducer.test.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import reducer from '../../redux/reducers/articleReducer';
+import * as articleTypes from '../../redux/actions-types/articleTypes';
+import { articleData } from '../../__mocks__/dummyData';
+
+import store from '../../redux/store';
```
Comment (Nziranziza) [inline src/__tests__/reducers/articleReducer.test.js]: Why not use mocking store?

**Review by mcaleb808 — APPROVED** [review]

File: src/__tests__/reducers/articleReducer.test.js (hunk 1 above)
Comment (oesukam) [reply src/__tests__/reducers/articleReducer.test.js]: @Nziranziza It is for testing the reducer, not the component

**Comment by oesukam** [comment]
I am fixing tests after rebasing

File: src/__tests__/reducers/articleReducer.test.js (hunk 1 above)
Comment (oesukam) [reply src/__tests__/reducers/articleReducer.test.js]: @Nziranziza Your feedback has helped me changed it to the initial state instead of mocking. Thanks.

**Review by Nziranziza — APPROVED** [review]

**Comment by oesukam** [comment]
@kimotho-njoki I have fixed conflicts and fixed tests. The PR is ready to be merged

**Pull request merged**